### PR TITLE
Improve Patient Record Found Dialog

### LINKED ISSUE #4701: Patient creation pop-up buttons spacing are not consistent
**Describe the bug**

the buttons in the patient creation pop-up button spacing are not consistent. ( responsive view )

**To Reproduce**

https://care.coronasafe.in/facility/4e12d77a-803c-45a0-ae7f-f1d4080a6d1f/patient

**Expected behavior**

the button in the patient creation pop-up should be having consistent spacing ( Responsive view )

**Screenshots**

image

### PR DESCRIPTION
**Pull request by Ashesh3**
Fixes #4701 

- Removes material-ui imports
- Fix spacing

image

image

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

### CODE REVIEW

File: src/Components/Facility/DuplicatePatientDialog.tsx [hunk 1]
```diff
@@ -27,99 +18,114 @@ const DuplicatePatientDialog = (props: Props) => {
...
-      <DialogContent>
-        <div className="grid gap-4 grid-cols-1">
-          <div>
-            <p className="leading-relaxed text-sm">
-              It appears that there are patient records that contain the same
-              phone number as the one you just entered. (
-              <span className="font-bold">{patientList[0].phone_number}</span>)
-            </p>
+    <DialogModal
+      title={<FieldLabel className="text-lg">Patient Records Found</FieldLabel>}
```
Comment (rithviknishad) [inline src/Components/Facility/DuplicatePatientDialog.tsx]: @Ashesh3 the DialogModal already applies styling for the title. Was the title styling changed intentionally?
```suggestion
      title="Patient Records Found"
```

**Review by rithviknishad — APPROVED** [review]

File: src/Components/Facility/DuplicatePatientDialog.tsx (hunk 1 above)
Comment (Ashesh3) [reply src/Components/Facility/DuplicatePatientDialog.tsx]: @rithviknishad I didn't double check it, just took the template from `FacilitiesSelectDialogue.tsx` https://github.com/coronasafe/care_fe/blob/develop/src/Components/ExternalResult/FacilitiesSelectDialogue.tsx#L21
I'll go ahead and remove it here.

**Comment by nihal467** [comment]
@Ashesh3 
image

the new UI doesn't look good to me with the new blue radio button, marking it for discussion on EOD

**Comment by rithviknishad** [comment]
@Ashesh3 could you try setting `accent-[color]` class?

**Comment by Ashesh3** [comment]
Fixed:
SecondhandAmbushbug

**Comment by Ashesh3** [comment]
> @Ashesh3 could you try setting `accent-[color]` class?

Didn't have any effect here, but `text-[color]` worked.

**Comment by nihal467** [comment]
LGTM

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**